models/uniswap/base/uniswap_v3_base_trades.sql

```diff
@@ -0,0 +1,108 @@
+{{ config(tags=['dunesql'],
+    schema = 'uniswap_v3_base',
+    alias = alias('trades'),
+    partition_by = ['block_date'],
```

Comment:
You can add a new column `block_month` for partition.
```suggestion
    partition_by = ['block_month'],
```